src/dlfcn.c

```diff
@@ -689,7 +699,18 @@ static BOOL is_valid_address( const void *addr )
  */
 static BOOL is_import_thunk( const void *addr )
 {
+#if defined _M_ARM64 || defined __aarch64__

```

Comment:
Code style in this file is:
```suggestion
#if defined(_M_ARM64) || defined(__aarch64__)
```